mesheryctl/internal/cli/root/components/list.go

```diff
@@ -40,39 +38,15 @@ mesheryctl components list --page 2
 // To view the number of components present in Meshery
 mesheryctl components list --count
 	`,
-	PreRunE: func(cmd *cobra.Command, args []string) error {
-		// Check prerequisites for the command here
-
-		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
-		if err != nil {
-			return err
-		}
-		err = utils.IsServerRunning(mctlCfg.GetBaseMesheryURL())
-		if err != nil {
-			return err
-		}
-		ctx, err := mctlCfg.GetCurrentContext()
-		if err != nil {
-			return err
-		}
-		err = ctx.ValidateVersion()
-		if err != nil {
-			return err
-		}
-		return nil
-	},
 	RunE: func(cmd *cobra.Command, args []string) error {
-		if len(args) != 0 {
-			return errors.New(utils.SystemModelSubError("this command takes no arguments\n", "list"))
-		}
 		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
 		if err != nil {
-			log.Fatalln(err, "error processing config")
+			utils.Log.Error(err)
+			return nil
```

Comment:
Wouldn't we want to report this failure to the user and ask if they would like generate a new config? We have an input prompt on this question in another area already that could be used.